nuts/context.py

```diff
@@ -19,9 +20,10 @@ class NutsContext:
         i.e. the yaml file that is converted to nuts tests
     """
 
-    def __init__(self, nuts_parameters: Any = None):
+    def __init__(self, nuts_parameters: Any = None, pytestconfig: Config = None):
```

Comment:
Now every context needs to have the pytestconfig but that's maybe anyway good for future context implementations

Reply:
Yes, this was part of my thoughts, that probably other contexts will also need this feature to pass on parameters in the future. So I integrated it into the NutsContext directly.